src/manifest/manifest_schema.ts

```diff
@@ -1,9 +1,7 @@
 import { ISlackDatastore } from "../datastore/types.ts";
 import { ISlackFunctionDefinition } from "../functions/types.ts";
-import type {
-  ParameterDefinition,
-  ParameterSetDefinition,
-} from "../parameters/mod.ts";
+import type { ParameterSetDefinition } from "../parameters/mod.ts";
```

Comment:
ultra nit: The fact that we didn't move this to `parameters/types.ts` as well sticks out to me here

Reply:
Done in 54e5989